src/services/api/reviews/useUserReviews.ts

```diff
@@ -73,3 +75,11 @@ export const useUserMultipleReviews = ({
     isError: queryInfo.some((query) => query.isError),
   };
 };
+
+export const useHasExceededMaxActiveReviews = (userId: number | undefined) => {
+  const { data: allReviews = { data: [] } } = useUserReviews({
+    userId,
+  });
+  const activeReviews = allReviews?.data?.filter(isReviewActive);
+  return activeReviews.length >= config.max_active_reviews;
```

Comment:
I see that this code is repeated here
https://github.com/bitcointranscripts/transcription-review-front-end/blob/1eec8c58b858934763f309c8d1acc894acf5c3d8/src/components/tables/EditableTranscriptsTable.tsx#L44-L47

What's the deal with the `EditableTranscriptsTable`? I don't see it being imported anywhere.

Also, I understand that the `useUserReviews` hook accepts a `status`, therefore we could just ask for the `active` reviews instead of getting all the reviews and then filter.

Reply:
The`EditableTranscriptsTable` was a generic table we used previously for the Transcripts. I think this was refactored and not removed from the codebase. I would raise another PR to remove it.